### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by notwidow** [ryanoasis/nerd-fonts#1014] [Q&A]
Can i know why are there releases in the nerd fonts? .. might be stupid question but i just figured it out and want to know. Thanks

**Comment by Finii** [comment]
I guess I dont understand the question.

New release means bugs fixed, symbols added,...

**Reply by notwidow** [reply]
> I guess I dont understand the question.
> 
> New release means bugs fixed, symbols added,...

i also use nerd fonts. in gnome desktop some icons are shown correctly but the ones shown in gnome are not shown in my qtile setup.. can you tell me does it have to do something with my setup?

**Reply by Finii** [reply]
Probably. I dont know what `qtile` is, though 😬

**Reply by notwidow** [reply]
> Probably. I dont know what `qtile` is, though 😬

forget qtile. i use neovim and there are some icons which get shown in gnome but some of those icons dont get shown in kde.

**Reply by Finii** [reply]
Real `neovim`? Then it solely depends on your terminal application (i.e. Gnome Terminal, Kitty, Tillix, ...)

**Reply by notwidow** [reply]
> Real `neovim`? Then it solely depends on your terminal application (i.e. Gnome Terminal, Kitty, Tillix, ...)

not even neovim but some other icons also. some render in kde while some render in gnome..

**Reply by Finii** [reply]
Then it must be a problem with your setup and not the font itself?

**Reply by notwidow** [reply]
> Then it must be a problem with your setup and not the font itself?

im asking that from you... why some icons do not show up in kde while the show up in gnome with same.

**Reply by Finii** [reply]
Maybe you should raise a proper issue then and not start with a completely unrelated question?

**Comment by notwidow** [comment]
i use alacritty and sometimes kitty
